client/src/components/Login.js

```diff
@@ -66,15 +76,17 @@ class Login extends Component {
           <form>
             <Grid container direction='column' spacing={4} justify='center'>
               <Grid item>
-                <Typography variant='h4'>Welcome back!</Typography> 
+                <Typography variant='h4' style={FormStyles.title}>Welcome back!</Typography> 
```

Comment:
Needs more consistency on how the components are styled, otherwise it might get confusing as the app gets bigger. Check out my message on the material-ui channel on slack